src/store/router/actions.test.js

```diff
@@ -17,25 +23,43 @@ describe('Router actions', () => {
   })
 
   test('should get the pool', async () => {
-    await getPool('0xef95192CC1B7766A06629721Cf8C7169ed34810a')(dispatchMock)
-    console.log(dispatchMock.mock.calls[1][0])
+    await getPool(TEST_TOKEN)(dispatchMock)
     expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
     expect(dispatchMock.mock.calls[1][0].type).toBe(Types.GET_POOL)
   })
 
   test('should get token count', async () => {
     await getTokenCount()(dispatchMock)
-    console.log(dispatchMock.mock.calls[1][0])
     expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
     expect(dispatchMock.mock.calls[1][0].type).toBe(Types.GET_TOKEN_COUNT)
   })
 
   test('should get total poled', async () => {
     await getTotalPooledValue()(dispatchMock)
-    console.log(dispatchMock.mock.calls[1][0])
     expect(dispatchMock.mock.calls[1][0].payload).not.toBeUndefined()
     expect(dispatchMock.mock.calls[1][0].type).toBe(
       Types.GET_TOTAL_POOLED_VALUE,
     )
   })
+
```

Comment:
in my case code that I don't use I don't put, then it is more readable and I avoid to have code that maybe I will never use

Reply:
I will likely be using that code, didn't want to remove it until I was sure. Working on it now!